src/lib.rs

```diff
@@ -48,6 +47,11 @@ impl BeforeMiddleware for PostgresMiddleware {
         req.extensions.insert::<PostgresMiddleware>(Value(self.pool.clone()));
         Ok(())
     }
+    
+    fn catch(&self, _: &mut Request, err: IronError) -> IronResult<()> {
+        println!("catch called");
```

Comment:
Sorry for the impromptu code review, I was just interested in your fix for SslMode. This println probably shouldn't be merged into master :)